Multisig/Logic/Models/PrivateKey/web3swift/BIP32HDNode.swift

```diff
@@ -0,0 +1,303 @@
+//
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import Foundation
+import BigInt
+import CryptoSwift
+
+// https://github.com/matter-labs/web3swift
+
+extension UInt32 {
+    public func serialize32() -> Data {
+        var bigEndian = self.bigEndian
+        let count = MemoryLayout<UInt32>.size
+        let bytePtr = withUnsafePointer(to: &bigEndian) {
+            $0.withMemoryRebound(to: UInt8.self, capacity: count) {
+                UnsafeBufferPointer(start: $0, count: count)
+            }
+        }
+        let byteArray = Array(bytePtr)
+        return Data(byteArray)
+    }
+}
+
+public class HDNode {
+    public struct HDversion{
+        public var privatePrefix: Data = Data(hex: "0x0488ADE4")
+        public var publicPrefix: Data = Data(hex: "0x0488B21E")
+        public init() {
+
+        }
+    }
+    public var path: String? = "m"
```

Comment:
Why "m" ?, can we use meaning and avoid constants

Reply:
I think this is part of the standard. This code is copied, so I will not change it here.